tests/test_federation.py

```diff
@@ -276,3 +276,111 @@ def test_cross_signing_keys_retry(self):
             "ed25519:" + remote_self_signing_key in self_signing_key["keys"].keys(),
         )
         self.assertTrue(remote_self_signing_key in self_signing_key["keys"].values())
+
+
+class StripUnsignedFromEventsTestCase(MessageAcceptTests):
+    def setUp(self):
+        super().setUp()
+
+    def strip_unauthorized_unsigned_values(self):
```

Comment:
Can you check that these tests get run by inspecting the `trial` output? IIRC only methods whose name starts with `test` are actually considered as tests to be run.

Reply:
They do run but only if you run them individually, which I can now see how that would be a problem. I didn't realized I needed to add `test` to the beginning of the method, thank you!